java/src/main/native/src/row_conversion.cu

```diff
@@ -14,46 +14,86 @@
  * limitations under the License.
  */
 
+#include <algorithm>
```

Comment:
Absolute nitpick: It's possible that the header order might run afoul of the (new?) prescription: "Nearest first".

Apropos, it might be worth checking if `"row_conversion.hpp"` should not come first. Then `cudf/*`, then `rmm/`, `thrust/`, and finally the standard library headers.

Reply:
Done